app/kubernetes-ingress-controller/troubleshooting/kubernetes-api-server.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Debugging Kubernetes API Server connectivity
+short_title: Kubernetes API Server
+
+description: |
+  How do I customize {{ site.kic_product_name }}'s connection to the Kubernetes API Server?
+
+content_type: reference
+layout: reference
+
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - index: kubernetes-ingress-controller
+    section: Troubleshooting
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+---
+
+## Authentication to the Kubernetes API Server
+
+A number of components are involved in the authentication process and the first step is to narrow down the source of the problem, namely whether it is a problem with service authentication or with the kubeconfig file.  Both authentications must work:
+
+{% mermaid %}
+graph RL
+    ingress["ingress controller"]
+    apiserver["apiserver"]
+
+    ingress -- "service authentication" --> apiserver
+{% endmermaid %}
+
+## Service authentication
+
+The Ingress controller needs information from API server to configure Kong.  Therefore, authentication is required, which can be achieved in two different ways:
+
+1. **Service Account**: This is recommended because nothing has to be configured.  The Ingress controller will use information provided by the system to communicate with the API server.  See 'Service Account' section for details.
```

Comment:
```suggestion
1. **Service Account**: This is recommended because nothing has to be configured.  The Ingress controller will use information provided by the system to communicate with the API server. See the [Service Account](#service-account) section for details.
```